docs/ides/gateway.md

```diff
@@ -74,3 +74,44 @@ explaining this IDE specification.
    ![Gateway IDE Opened](../images/gateway/gateway-intellij-opened.png)
 
 > Note the JetBrains IDE is remotely installed into `~/.cache/JetBrains/RemoteDev/dist`
+
+## Configuring Gateway to use self-signed certificates
+
+When attempting to connect to a Coder deployment that uses self-signed certificates,
+you may receive the following error in Gateway:
+
+```console
+Failed to configure connection to https://coder.internal.enterprise/: PKIX path building failed: sun.security.provider.certpath.SunCertPathBuilderException: unable to find valid certification path to requested target
+```
+
+To resolve this issue, you will need to add Coder's certificate to the Java trust store
+present on your local machine. Here is the default location of the trust store for
+each OS:
+
+```console
+# Linux
+$JAVA_HOME/lib/security/cacerts
+<Gateway installation directory>/etc/pki/java/cacerts
+<Gateway installation directory>/etc/ssl/certs/java/cacerts
+
+# macOS
+$<Gateway installation directory>/lib/security/cacerts
+$<Gateway installation directory>/jre/lib/security/cacerts
+
+# Windows
+C:\Program Files (x86)\<Gateway installation directory>\jre<version>\lib\security\cacerts
```

Comment:
If installed via JetBrains Toolbox, the path is,
`C:\Users\<YOUR USER>\AppData\Local\JetBrains\Toolbox\apps\Gateway`